_posts/2020-08-08-presto-eighth-birthday.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Happy Eighth Birthday Presto!"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+Today, Presto turned eight years old! As Presto co-creator
+Dain Sundstrom points out, there’s a reason why the eighth birthday is a
+little special:
+
+https://twitter.com/daindumb/status/1292296395219595264
```

Comment:
I would just use an image and get rid of the jekyll plugin for now .. unless we plan to do lots more twitter embedding.

Reply:
I think I am planning on it. Typically with announcements or promoting individuals within the community, I like linking individual's tweets about their work, as well as, linking their GitHub accounts.